Dockerfile

```diff
@@ -33,10 +33,11 @@ ENV MOJO_REVERSE_PROXY 0
 
 # Do not change these variables unless you know what you're doing
 ENV CONVOS_HOME /data
-ENV MOJO_LISTEN http://*:3000
+ENV MOJO_LISTEN http://*:$port
 ENV MOJO_MODE production
 
 VOLUME ["/data"]
-EXPOSE 3000
-CMD []
-ENTRYPOINT ["/convos-stable/script/convos", "daemon"]
+EXPOSE $port
+
+CMD ["daemon"]
+ENTRYPOINT ["/convos-stable/script/convos"]
```

Comment:
Cool! Didn't know CMD worked like this. I thought it was an alternative to ENTRYPOINT.